src/kinsol/kinsol_io.c

```diff
@@ -67,6 +67,28 @@ int KINSetUserData(void* kinmem, void* user_data)
   return (KIN_SUCCESS);
 }
 
+/*
+ * -----------------------------------------------------------------
+ * Function : KINSetOwnUserData
+ * -----------------------------------------------------------------
+ */
+
+int KINSetOwnUserData(void* kinmem, sunbooleantype own_user_data)
```

Comment:
Like IDA, KINSOL is inconsistent about this but to follow the general convention prefix with `kin`
```suggestion
int kinSetOwnUserData(void* kinmem, sunbooleantype own_user_data)
```